### PR TITLE
Api 2.6.0

### PR DESCRIPTION
**Pull request by AviSander**
API 2.6 release

### CODE REVIEW

**Comment by AviSander** [comment]
this should be the last update to the 'old toolset'...

File: templates/visuals/.api/v2.3.0/schema.capabilities.json [hunk 1]
```diff
@@ -51,6 +51,10 @@
         "supportsLandingPage": {
             "type": "boolean",
             "description": "Indicates whether the visual supports a landing page"
+        },
```
Comment (uve) [inline templates/visuals/.api/v2.3.0/schema.capabilities.json]: please remove "subtotals" from API 2.3

File: templates/visuals/.api/v2.6.0/schema.capabilities.json [hunk 2]
```diff
@@ -0,0 +1,1193 @@
...
+                },
+                {
+                    "required": [
+                        "operations"
+                    ]
+                }
+            ]
+        }
+    }
+}
```
Comment (uve) [inline templates/visuals/.api/v2.6.0/schema.capabilities.json]: Add **multiselectsupport** flag (From Ramiz)

File: templates/visuals/.api/v2.3.0/schema.capabilities.json (hunk 1 above)
Comment (AviSander) [reply templates/visuals/.api/v2.3.0/schema.capabilities.json]: fixed

File: templates/visuals/.api/v2.6.0/schema.capabilities.json (hunk 2 above)
Comment (AviSander) [reply templates/visuals/.api/v2.6.0/schema.capabilities.json]: multiSelect support is not part of 2.6. It'll go as part of 3.1.

**Pull request merged**